.github/workflows/_cdk.yml

```
@@ -55,10 +56,11 @@
 
       - name: Set variables and version for prod
         if: startsWith(github.ref, 'refs/tags')
-        run: |
+        run: | 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)